src/tellor_disputables/data.py

```diff
@@ -246,9 +264,9 @@ async def parse_new_report_event(event: AttributeDict[str, Any], web3: Web3, con
         status_str = disputable_str(disputable, query_id)
 
         return NewReport(
-            chain_id=web3.eth.chain_id,
+            chain_id=endpoint.eth.chain_id,
             eastern_time=args["_time"],
-            tx_hash=tx_hash.hex(),
+            tx_hash=tx_hash,
```

Comment:
Why not `.hex()`?

Reply:
Since the NewEvent parser function `parse_new_report_event` now takes a tx hash as input instead of getting one from an event `AttributeDict`, it's no longer necessary to cast the `tx_hash` to hex